packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -74,8 +75,17 @@ async function handleDeleteContext(contextName: string) {
       aria-label="No Resource Panel"
       icon={EngineIcon}
       title="No Kubernetes contexts found"
-      message="Check that $HOME/.kube/config exists or KUBECONFIG environment variable has been set correctly."
-      hidden={$kubernetesContexts.length > 0} />
+      message="Check that Kubernetes context is created and selected. This can be done in Resources page."
```

Comment:
Maybe instead of "This can be done in Resources page", something like "you can create local Kubernetes cluster from Podman Desktop"